src/ForkCMS/Bundle/InstallerBundle/Resources/views/Installer/form_theme.html.twig

```diff
@@ -202,7 +202,7 @@
     {% if form.parent %}<span class="help-block errorMessage">{% else %}<div class="alert alert-danger errorMessage">{% endif %}
     <ul class="list-unstyled">
 {%- for error in errors -%}
-  <li><span class="fa fa-exclamation-circle"></span> {{ error.message }}</li>
+  <li><span class="fa fa-exclamation-circle" aria-hidden="true"></span> {{ error.message }}</li>
```

Comment:
can't you use the macro icon here?

Reply:
I didn't do it because the macro files are bound with the backend and the frontend, and this is the installer bundle, it is something that stands apart so I didn't use it here. If you think I should use a macro here: do I use the back or frontend one?

Reply:
Mmh ok i understand why you didn't you it, i am not sure anymore if you should change it, maybe not. But if we change it, I should use the backend one